pandas/plotting/_core.py

```diff
@@ -228,6 +228,8 @@ def hist_frame(
     Returns
     -------
     matplotlib.AxesSubplot or numpy.ndarray of them
+        An ndarray is returned with one :class:`matplotlib.AxesSubplot`
+        per column.
```

Comment:
Thank you for fixing the style of the docstring @erichxchen. 
I think think needs a bit more of work, as this doesn't seem very accurate. In general this will return the matplotlib object representing the histogram. This is what we should say first. Then, I'm not sure if an array is returned when `by` is used. Can you do some tests and make sure that what we provide here is relevant and correct please?

Reply:
Sure, I will be working on this